forgevtt-module.js

```diff
@@ -1791,40 +1786,65 @@ class ForgeAPI {
 }
 
 class ForgeCompatibility {
+
+    /**
+     * The global isNewerVersion will be removed in v14, so we need a utility function to alias to whichever is available.
+     * @param {string} version The version to check
+     * @param {string} target The version to check against
+     * @returns {boolean} True when the version is newer than the target, false otherwise
+     */
+    static _isNewerVersion(version, target) {
+        try {
```

Comment:
Is there a risk that we could have an actual, meaningful failure from one of the `isNewerVersion` methods?

If so, maybe we consider some optional chaining here to look for `foundry.utils.isNewerVersion`?

Reply:
This code is moved from its old home in the `ForgeVTT` class. When I was working on that originally, (https://github.com/ForgeVTT/fvtt-module-forge-vtt/pull/37#discussion_r1636952503) I found that referencing `foundry` at all would throw an error on older versions where it wasn't defined. I think this is the same as the problems you recently ran into while working on the file picker.